doc/src/cli/shorthand.md

```diff
@@ -21,7 +20,7 @@ Usage: gs branch (b) create (c) [<name>] [flags]
 {gray}# ...{reset}
 ```
 
-The shorthand is built by joining the bits in parentheses.
+The shorthand for a command is the bits in parentheses joined together.
 For example, the shorthand for $$gs branch create$$ above is `gs bc`.
```

Comment:
Why using $$ ?

Isn't the LaTeX syntax operator?

I would have expected 


```suggestion
 For example, the shorthand for |gs branch create` above is `gs bc`.
```

Reply:
`$$` is a LaTeX syntax, but it's not really supported in Markdown—unless you specifically enable it.
Since it's not part of the Markdown standard, but is obvious and familiar enough, it made sense to use it for a custom extension. I have a custom hook installed into mkdocs that will turn `$$gs foo bar$$` into a link to the CLI reference, and `$$spice.foo.bar$$` into a link to the configuration reference.